bindgen/TypeTranslator.cpp

```diff
@@ -138,6 +147,10 @@ std::shared_ptr<Type> TypeTranslator::translate(const clang::QualType &qtpe,
             std::make_shared<PrimitiveType>("Byte"), sizeInBits / 8);
     }
 
+    if (tpe->isFunctionType()) {
+        return nullptr;
```

Comment:
No, it always used to return `nullptr` for function type, I just made it explicit.

I found this declaration in libio.h:
```c
typedef __ssize_t __io_read_fn (void *__cookie, char *__buf, size_t __nbytes);
```
That caused an error because `TypeDef` was created with `nullptr`.

Actually I am not aware of where function types may be used and what should we do with them. Maybe we should create an issue for investigating it.